tox.ini

```diff
@@ -1,17 +1,23 @@
 [tox]
-envlist = py37,py38
+envlist = py37,py38,py39,py310
 
 [testenv]
-extras = dev
-commands=
+extras =
+    dev
+deps =
+    flake8
+    black
+;    mypy
+commands =
     python -m pytest --cov stac_pydantic --cov-report xml --cov-report term-missing
-deps=
-    numpy
+    flake8 .
+    black .
+;    mypy stac_pydantic/
 
 # Lint
 [flake8]
-exclude = .git,__pycache__,docs/source/conf.py,old,build,dist
-max-line-length = 90
+exclude = .git,__pycache__,docs/source/conf.py,old,build,dist,.venv,.tox,.idea
+max-line-length = 120
```

Comment:
Regardless of the value here, `black` defaults to `88`, so they suggest setting the flake8 config to 88: https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#line-length

Reply:
I was wondering about that -- black wasn't giving an error but flake8 was on those lines. I've updated the flake8 config per the black documentation.